VectorValued.pm

```diff
@@ -14,20 +14,69 @@ use PDL;
 use PDL::Exporter;
 use PDL::VectorValued::Utils;
 our @ISA = qw(PDL::Exporter);
-our @EXPORT_OK =
-  (
-   (@PDL::VectorValued::Utils::EXPORT_OK), ##-- inherited
-   qw(vv_uniqvec),
-   qw(rleND rldND),
-   qw(vv_indx),
-  );
+
+our (@EXPORT_OK);
+BEGIN {
+  ##--------------------------------------------------------------------
+  ## Conditional bindings for PDL > v2.079
+  ##  + see https://github.com/moocow-the-bovine/PDL-VectorValued/issues/5
+
+  ## @VV_SYMBOLS : exportable symbols (vv_FOO)
+  my @VV_SYMBOLS =
+    (
+     (@PDL::VectorValued::Utils::EXPORT_OK), ##-- inherited
+     qw(vv_uniqvec),
+     qw(vv_rleND vv_rldND),
+     qw(vv_indx),
+    );
+
+  # %VV_IMPORT: we expect these symbols to be defined by PDL core
+  my %VV_IMPORT =
+    (map {($_=>undef)}
+      (version->parse($PDL::VERSION) > version->parse('2.079')
+       ? qw(cmpvec eqvec vsearchvec)
+       : qw()));
```

Comment:
@mohawk2 should I just get rid of this (and the membership check at line 59)?

Reply:
Please note the above thoughts are somewhat superseded by my most recent comment on #5 - sorry for any confusion resulting.